src/DebugSerial.cpp

```diff
@@ -1,11 +1,11 @@
 /**
- * @file debugSerial.cpp
+ * @file DebugSerial.cpp
```

Comment:
why are some file names capitalized and others aren't?

Reply:
C++ Classes and their header files are usually capitalised while C header files are not. 
The only grey area is SysPara.h because it doesn't have a separate implementation file.

Reply:
I was thinking about that but e.g. in this file, no classes are defined

Reply:
Fair enough! There shouldn't be a separate implementation file in that case, however.